recipes/hole-filling-liepa/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "hole-filling-liepa" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: {{ version }}
+
+source:
+  url: https://github.com/russelmann/hole-filling-liepa/archive/v{{ version }}.tar.gz
+  sha256: c73a65bd39b30801f5c85f7245de0f1ae8458d6eb78d657f63d8976d63cf4a47
+
+build:
+  number: 0
+  skip: true  # [py<38]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - make
+    - pip
+    - pybind11
+    - eigen
+  host:
+    - python
+    - pip
+    - numpy
```

Comment:
```suggestion
```
Do you actually link your C++ code against the NumPy C API? If not, then you don't need numpy in this section.

Reply:
I generate Python bindings using pybind11 which internally links to NumPy. Because of that building the project requires NumPy.

Reply:
Accidentally clicked on "resolve". I will fix this separately...